src/components/UriViewer/LogViewer.ts

```diff
@@ -0,0 +1,224 @@
+import _ from 'lodash/fp';
+import { useCallback, useEffect, useState } from 'react';
+import { div, h, span } from 'react-hyperscript-helpers';
+import { SimpleTabBar } from 'src/components/tabBars';
+import { isAzureUri } from 'src/components/UriViewer/uri-viewer-utils';
+import { Ajax } from 'src/libs/ajax';
+import { useCancellation, useOnMount, withDisplayName } from 'src/libs/react-utils';
+import { newTabLinkProps } from 'src/libs/utils';
+
+import { ButtonOutline } from '../common';
```

Comment:
Typically, Terra UI modules use absolute paths for imports. The only exceptions are when importing something from the same folder.